### PR TITLE
Add mode specific help text

### PR DESCRIPTION
**Pull request by nikhil550**
Moves the help text to the scriptUtils, and create mode specific help text.

For example:
```
Usage: 
  network.sh deployCC [Flags]

    Flags:
    -c <channel name> - Name of channel to deploy chaincode to
    -ccn <name> - Chaincode name. This flag can be used to deploy one of the asset transfer samples to a channel. Sample options: basic (default),ledger, private, sbe, secured
    -ccl <language> - Programming language of chaincode to deploy: go (default), java, javascript, typescript
    -ccv <version>  - Chaincode version. 1.0 (default), v2, version3.x, etc
    -ccs <sequence>  - Chaincode definition sequence. Must be an integer, 1 (default), 2, 3, etc
    -ccp <path>  - (Optional) File path to the chaincode. When provided, the -ccn flag will be used only for the chaincode name.
    -ccep <policy>  - (Optional) Chaincode endorsement policy using signature policy syntax. The default policy requires an endorsement from Org1 and Org2
    -cccg <collection-config>  - (Optional) File path to private data collections configuration file
    -cci <fcn name>  - (Optional) Name of chaincode initialization function. When a function is provided, the execution of init will be requested and the function will be invoked.

    -h - Print this message

 Possible Mode and flag combinations
   deployCC -ccn -ccl -ccv -ccs -ccp -cci -r -d -verbose

 Examples:
   network.sh deployCC -ccn basic -ccl javascript
   network.sh deployCC -ccn mychaincode -ccp ./user/mychaincode -ccv 1 -ccl javascript
```

Let me know what you think.

Signed-off-by: NIKHIL E GUPTA <ngupta@symbridge.com>

### CODE REVIEW

**Review by lehors — CHANGES_REQUESTED** [review]
I think if we go that way we have to do a bit more. I suggest a few more edits for consistency.

File: test-network/scriptUtils.sh [hunk 1]
```diff
@@ -6,6 +6,118 @@ C_GREEN='\033[0;32m'
...
+# Print the usage message
+function printHelp() {
+  USAGE="$1"
+  if [ "$USAGE" == "up" ]; then
+    println "Usage: "
+    println "  network.sh \033[0;32mup\033[0m [Flags]"
+    println
+    println "    Flags:"
+    println "    -ca <use CAs> -  create Certificate Authorities to generate the crypto material"
+    println "    -c <channel name> - channel name to use (defaults to \"mychannel\")"
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: Name of channel name to create (defaults to "mychannel")

File: test-network/scriptUtils.sh [hunk 2]
```diff
@@ -6,6 +6,118 @@ C_GREEN='\033[0;32m'
...
+    println "   \033[0;32mup createChannel\033[0m -ca -c -r -d -s -i -verbose"
+    println
+    println " Examples:"
+    println "   network.sh up createChannel -ca -c mychannel -s couchdb -i 2.0.0"
+  elif [ "$USAGE" == "createChannel" ]; then
+    println "Usage: "
+    println "  network.sh \033[0;32mcreateChannel\033[0m [Flags]"
+    println
+    println "    Flags:"
+    println "    -c <channel name> - channel name to use (defaults to \"mychannel\")"
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: Name of channel name to create (defaults to "mychannel")

File: test-network/scriptUtils.sh [hunk 3]
```diff
@@ -6,6 +6,118 @@ C_GREEN='\033[0;32m'
...
+function printHelp() {
+  USAGE="$1"
+  if [ "$USAGE" == "up" ]; then
+    println "Usage: "
+    println "  network.sh \033[0;32mup\033[0m [Flags]"
+    println
+    println "    Flags:"
+    println "    -ca <use CAs> -  create Certificate Authorities to generate the crypto material"
+    println "    -c <channel name> - channel name to use (defaults to \"mychannel\")"
+    println "    -s <dbtype> - the database backend to use: goleveldb (default) or couchdb"
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: done.

**Comment by nikhil550** [comment]
As suggested by Arnaud, I did some edits to the help text more broadly to clean up the language and the clarity.

I also took the opportunity to remove the "restart" mode, which is not used in any of the tutorials or samples, and does not have a practical use.

File: test-network/scriptUtils.sh [hunk 4]
```diff
@@ -6,6 +6,118 @@ C_GREEN='\033[0;32m'
...
+    println "    -r <max retry> - CLI times out after certain number of attempts (defaults to 5)"
+    println "    -d <delay> - delay duration in seconds (defaults to 3)"
+    println "    -i <imagetag> - the tag to be used to launch the network (defaults to \"latest\")"
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: Updated with clearer language throughout

File: test-network/scriptUtils.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (lehors) [reply test-network/scriptUtils.sh]: You have one too many "name" in here.

File: test-network/scriptUtils.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (lehors) [reply test-network/scriptUtils.sh]: same here.

**Comment by lehors** [comment]
> As suggested by Arnaud, I did some edits to the help text more broadly to clean up the language and the clarity.
> 
> I also took the opportunity to remove the "restart" mode, which is not used in any of the tutorials or samples, and does not have a practical use.

Sounds reasonable to me but would be worth adding that to the commit log.
I think restart could be useful if it retained all the arguments but as it is, I agree it's not really useful.

File: test-network/scriptUtils.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: done

File: test-network/scriptUtils.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikhil550) [reply test-network/scriptUtils.sh]: done.

**Review by lehors — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Pull request merged**